examples/weekly-summary-email/inngest.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Weekly summary email",
+  "id": "enough-boar-589f6a",
+  "triggers": [
```

```suggestion
{
  "name": "Weekly summary email",
  "description": "...",
  "tags": ["golang"],
  "id": "enough-boar-589f6a",
  "triggers": [
```

This needs `description` and `tags` fields adding for the site.

We could add a Dangerfile to prompt for this in the future.